src/iter/step_by.rs

```diff
@@ -0,0 +1,131 @@
+use super::plumbing::*;
```

Comment:
Rather than repeating `#[cfg(step_by)]`, you can add a single `#!` attribute here to cover the whole file:

```suggestion
#![cfg(step_by)]

use super::plumbing::*;
```